public/js/data.js

```diff
@@ -1,73 +1,34 @@
 if (sessionStorage.getItem('loggedIn') != "true") {location.href = "/admin-login";}
 
 $(document).ready( function () {
-
-    /* Levels chart */
-    var levZero = 0;
-    var levOne = 0;
-    var levTwo = 0;
-    var levThree = 0;
-
-    /* Store Type chart */
-    let small = 0;
-    let medium = 0;
-    let large = 0;
-    let convenience = 0;
-
-    levelArray = null;
-    typeArray = null;
-
-
-    if (!firebase.apps.length) {
-        firebase.initializeApp(config);
-    }
-
     google.charts.load('current', {'packages':['corechart']});
     
-    // Creates connection to database.
-    var db = firebase.database();
-
-    // Links to head of database.
-    var ref = db.ref("live_weller");
-
-    // Links to markets list.
-    var marketsRef = ref.child("markets");
-
-
-    // Call async function
-    callFirebase(loadCharts);
-
-
-
-    // Start of callback
-    function loadCharts() {
-        levelArray = [
-            ['Market Level', 'Number of Markets'],
-            ['Level 0', levZero],
-            ['Level 1', levOne],
-            ['Level 2', levTwo],
-            ['Level 3', levThree]
-        ];
+    let levelArray = null;
+    let typeArray = null;
+    
+    $.post('/data/general', {type: "general"}, function(data) {
```

Comment:
this can be a GET request since it's just retrieving data and `{ type }` isn't being used as far as I can tell

Reply:
I'll do this right now, forgot to take that little part out.  You'll have to approve again lol, sorry!